### PR TITLE
Update run-ansible.sh

### PR DESCRIPTION
**Pull request by carebare47**
## Proposed changes

Previously, aurora was using a system-installed non-conda ansible if it existed. We're in conda now, so let the os find the various ansible binaries via $PATH (which we know conda has modified by this point in the script)

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

**Review by niko-holmes — APPROVED** [review]

**Review by dg-shadow — APPROVED** [review]

File: ansible/roles/products/common/docker-container/tasks/modify-container.yml [hunk 1]
```diff
@@ -23,9 +23,25 @@
...
+
+- name: Please wait and don't close the empty terminator window while aurora is executing the modify container script with these commands {{ modify_script_commands }} with retries
   shell: "/tmp/{{ modify_script_name }}"
   become: yes
   changed_when: false
-  when: not skip_molecule_task|bool
+  retries: 10
+  delay: 3
+  register: result
+
```
Comment (rnzenha-s) [inline ansible/roles/products/common/docker-container/tasks/modify-container.yml:44]: Should we remove this empty line?

Comment (carebare47) [reply ansible/roles/products/common/docker-container/tasks/modify-container.yml:44]: c22e64dbca97864c7a5a87e055c85b958033d92c

**Comment by rnzenha-s** [comment]
This PR has been split into two: 
- https://github.com/shadow-robot/aurora/pull/622
- https://github.com/shadow-robot/aurora/pull/623

**Comment by niko-holmes** [comment]
Has been decomposed into https://github.com/shadow-robot/aurora/pull/622 and https://github.com/shadow-robot/aurora/pull/623

**Pull request closed without merging**